src/Symfony/Component/HttpFoundation/JsonResponse.php

```diff
@@ -59,6 +59,14 @@ public static function create($data = null, $status = 200, $headers = array())
     }
 
     /**
+     * Make easier the creation of JsonResponse from raw json
+     */
+    public static function fromJsonString($data = null, $status = 200, $headers = array())
```

Comment:
Np problem to switch to `createFromJsonString`. I'm used to remove the `create` prefix but if everyone prefer with it, I'm ok !